pystac/summaries.py

```diff
@@ -58,6 +58,12 @@ def __lt__(self: "T", other: "T") -> bool:
 
 
 class RangeSummary(Generic[T]):
+    minimum: T
+    """The minimum range value."""
```

Comment:
These doc strings seem redundant.  If the project style requires them, then definitely leave them.  Otherwise consider dropping them.  Without them, the code is more compact and to me just as readable.  But I'm fine with either way.

Reply:
The intention was to get these to show up in the API docs, but I now realize that we don't include `RangeSummary` in the API docs anyway. I'll remove these docstrings and add the `RangeSummary` classeto the API docs with an `:undoc-members` option so that these get included in the class docs.